### PR TITLE
feat(cleanup):[TRI-348] renamed PartTree

### PR DESCRIPTION
**Pull request by ds-ext-kmassalski**
- renamed without changing logic
- removed some not needed things

### CODE REVIEW

File: irs-api/src/main/java/net/catenax/irs/services/IIrsItemGraphQueryService.java [hunk 1]
```diff
@@ -20,7 +20,7 @@
 import net.catenax.irs.component.enums.JobState;
 
 /**
- * IrsPartTreeQueryService interface
+ * IIrsItemGraphQueryService interface
```
Comment (mkanal) [inline irs-api/src/main/java/net/catenax/irs/services/IIrsItemGraphQueryService.java:23]: You introduce the I prefix for Interface. This is quite ok but then it has to be consitent for all Interfaces. IMHO that is not the standard. Another approch is to use postfix for Impl like PartImpl only for the Implementation then the Interface must not have the I Prefix. Important is that we make it in a common way. If we have an agreement it make sense to make a common solution markes in the code standard : https://confluence.catena-x.net/display/ARTV/7.+IRS+Code+Standards
Interface have a Prefix I oder Implementation has a postfix Impl or another approach!

File: irs-api/src/main/resources/application.yml [hunk 2]
```diff
@@ -60,7 +60,6 @@ springdoc:
 
 irs:
   apiUrl: http://irs.int.demo.catena-x.net
-  partsTreeMaxDepth: 100
```
Comment (mkanal) [inline irs-api/src/main/resources/application.yml:63]: Why is this removed?

File: irs-api/src/main/java/net/catenax/irs/configuration/IrsConfiguration.java [hunk 3]
```diff
@@ -29,8 +29,4 @@ public class IrsConfiguration {
      */
     private URL apiUrl;
 
-    /**
-     * The maximum depth at which parts tree are recursively retrieved.
-     */
-    private int partsTreeMaxDepth = Integer.MAX_VALUE;
```
Comment (mkanal) [inline irs-api/src/main/java/net/catenax/irs/configuration/IrsConfiguration.java:35]: Why is that removed?

File: irs-api/src/main/resources/application.yml (hunk 2 above)
Comment (ds-ext-kmassalski) [reply irs-api/src/main/resources/application.yml:63]: Parameter was not used anywhere in the code.
Depth is handled/validated inside RegisterJob class.

File: irs-api/src/main/java/net/catenax/irs/configuration/IrsConfiguration.java (hunk 3 above)
Comment (ds-ext-kmassalski) [reply irs-api/src/main/java/net/catenax/irs/configuration/IrsConfiguration.java:35]: Parameter was not used anywhere in the code.
Depth is handled/validated inside RegisterJob class.

File: irs-api/src/main/java/net/catenax/irs/services/IIrsItemGraphQueryService.java (hunk 1 above)
Comment (ds-ext-kmassalski) [reply irs-api/src/main/java/net/catenax/irs/services/IIrsItemGraphQueryService.java:23]: I didnt change the interface name here, just JavaDoc comment. IIrsItemGraphQueryService name was there before.

**Review by ds-jhartmann — APPROVED** [review]
Maybe check, if the PUMLs are used on confluence and update them accordingly.
